transformers/storage/repository_test.go

```diff
@@ -795,9 +796,19 @@ func insertFlopKicks(blockNumber int64, kicks string, contractAddressID int64, d
 func insertTend(blockNumber int64, bidID string, contractAddressID int64, db *postgres.DB) {
 	headerID := insertHeader(db, blockNumber)
 	tendLog := test_data.CreateTestLog(headerID, db)
-	_, err := db.Exec(`INSERT into maker.tend (header_id, bid_id, lot, bid, address_id, log_id)
-		VALUES($1, $2::NUMERIC, $3::NUMERIC, $4::NUMERIC, $5, $6)`,
-		headerID, bidID, 0, 0, contractAddressID, tendLog.ID,
+	tendLog.Log.Topics = []common.Hash{
+		common.HexToHash(constants.TendSignature()),
+		common.HexToHash("0x0000000000000000000000003a673843d27d037b206bb05eb1abbc7288d95e66"),
+		common.HexToHash("0x000000000000000000000000000000000000000000000000000000000000000a"),
+		common.HexToHash("0x000000000000000000000000000000000000000000000000000007bb0f7b0800"),
+	}
```

Comment:
Depending on when this gets merged, I think you might be able to use a helper created by Eric to build a test log based off of test data (and assign events that way) - https://github.com/makerdao/vdb-mcd-transformers/pull/236/files#diff-eaee5d9c3bd36e4c40f28c33bad51985R97